flygym/fly.py

```diff
@@ -297,10 +301,24 @@ def __init__(
             If True, colored spheres will be added to the model to indicate
             the positions of the cameras (for vision) and odor sensors. By
             default False.
-        head_stabilization_kp : float
-            Position gain of the head stabilization actuators, by default
-            1000.0. Set to 0 to disable head stabilization.
+        neck_kp : float, optional
+            Position gain of the neck position actuators. If supplied, this
+            will overwrite ``actuator_kp`` for the neck actuators.
+            Otherwise, ``actuator_kp`` will be used.
         """
+        # Check neck actuation if head stabilization is enabled
+        if head_stabilization_model is not None:
+            if "joint_Head_yaw" in actuated_joints or "joint_Head" in actuated_joints:
```

Comment:
Would it not be possible for the author to override head stabilisation when supplying an action involving the head ? Or summing up the contribution ?

Reply:
Yes, I would prefer the module to yell at the user if this attempted. The user can then implement the override logic (or otherwise) explicitly. I think in general having the code doing something that's not the most obvious thing that it should do is not the best practice, especially since so far we have no demo case of overriding head stabilization or summing up the contributions.